mock/tests/e2e/App.spec.ts

```diff
@@ -54,29 +55,147 @@ test("after I type into the input box, its text changes", async ({ page }) => {
 });
 
-test("after I click the button, its label increments", async ({ page }) => {
-  await page.goto("http://localhost:8000/");
+test("after I click the button with a bad command, I get feedback that that command is bad", async ({
+  page,
+}) => {
+  //await page.goto("http://localhost:8000/");
   await page.getByLabel("Login").click();
+  await page.getByLabel("Command input").fill("badinput.csv");
   await page.getByLabel("Submit button").click();
-  await expect(page.getByLabel("Submit button")).toHaveValue(
-    "Clicked 1 times!"
+  const mock_input = "badinput.csv";
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "Command " + mock_input + " not found"
   );
-  // TODO WITH TA: Fill this in to test your button counter functionality!
 });
 
-test("after I click the button, my command gets pushed", async ({ page }) => {
-  await page.goto("http://localhost:8000/");
+test("after I enter load, my csv gets loaded", async ({ page }) => {
+  //await page.goto("http://localhost:8000/");
   await page.getByLabel("Login").click();
-  await page.getByLabel("Command input").fill("Awesome command");
+  const mock_input = "load strings.csv false";
+  await page.getByLabel("Command input").fill(mock_input);
   await page.getByLabel("Submit button").click();
-  const mock_input = `Awesome command`;
-  await expect(page.getByLabel("repl-history")).toHaveValue(mock_input);
-  // TODO: Fill this in to test your button push functionality!
+
+  await expect(page.getByLabel("repl-history")).toContainText(
+    "Loading file at destination"
+  );
+});
+
+test("I can load and view a mocked csv", async ({ page }) => {
+  await page.getByLabel("Login").click();
+  const loadCommand = "load strings.csv false";
+  await page.getByLabel("Command input").fill(loadCommand);
+  await page.getByLabel("Submit button").click();
+  await page.getByLabel("Command input").fill("view");
+  await page.getByLabel("Submit button").click();
+
+  // Assumes that if first two elems of csv were displayed, csv was displayed
+  await expect(page.getByLabel("repl-history")).toContainText("The");
+  await expect(page.getByLabel("repl-history")).toContainText("song");
+});
+
+test("I can call search after loading, and get the first row of the table", async ({
+  page,
+}) => {
+  await page.getByLabel("Login").click();
+  const loadCommand = "load strings.csv false";
+  await page.getByLabel("Command input").fill(loadCommand);
+  await page.getByLabel("Submit button").click();
+  const searchCommand = "search song";
+  await page.getByLabel("Command input").fill(searchCommand);
+  await page.getByLabel("Submit button").click();
+
+  // Assumes that if first two elems of csv were displayed, csv was displayed
+  await expect(page.getByLabel("repl-history")).toContainText("The");
+  await expect(page.getByLabel("repl-history")).toContainText("song");
+});
+
+test("After loading and viewing a csv, I can load and view a different csv", async ({
```

Comment:
Consider testing searching as well after loading a csv, as well as interactions between logging out, logging back in, and viewing and/or searching